api/src/api/v1/serialisers/volunteer_log.ts

```diff
@@ -0,0 +1,3 @@
+import { identity } from 'ramda';
+
+export default identity;
```

Comment:
Is this a functional joke?

Reply:
No, it's meant as a layer of decoupling. At the moment, our serialisation of logs is just `a -> a`. But if that ever changes it's gonna be easier to replace this function definition than it would be to replace two dozen calls to `Serialisers.identity`. Allows for more specific naming: `Serialisers.volunteerLog` tells you about context, rather than implementation (notwithstanding your comment above, obvs).